ddtrace/contrib/botocore/patch.py

```diff
@@ -147,12 +147,10 @@ def patched_api_call(original_func, instance, args, kwargs):
 
         region_name = deep_getattr(instance, "meta.region_name")
 
-        meta = {
-            "aws.agent": "botocore",
-            "aws.operation": operation,
-            "aws.region": region_name,
-        }
-        span.set_tags(meta)
+        span._set_str_tag("aws.agent", "botocore")
+        if operation is not None:
+            span._set_str_tag("aws.operation", operation)
+        span.set_tag("aws.region", region_name)
```

Comment:
`_set_str_tag`?

Reply:
I wasn't sure whether `region_name` can be assumed to be a string or not, but seems likely given the name :). Could be a good chance to use `maybe_stringify` some more! 😁

Reply:
yeah, I cannot imagine it being anything other than `"us-east1"` for example